apps/src/lib/levelbuilder/code-docs-editor/OrderableList.jsx

```diff
@@ -7,7 +7,8 @@ export default function OrderableList({
   list,
   setList,
   addButtonText,
-  renderItem
+  renderItem,
+  checkItemDeletionAllowed = item => true
```

Comment:
Can you tell me about how this works?

Reply:
Is this just setting a default if checkItemDeletionAllowed is not passed?

Reply:
Yes it's just a default. All items can be deleted unless a specific function is passed in to check.